.github/workflows/server-tests.yml

```diff
@@ -109,193 +40,36 @@ jobs:
           PR_NUMBER: ${{ github.event.number }}
           REPO_NAME: ${{ github.repository }}
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-     
-
   test:
-    name: Unit Tests
-    runs-on: ubuntu-latest
+    name: Tests
+    uses: frappe/frappe/.github/workflows/tests-base.yml@develop
```

Comment:
These files are now fetched from develop branch always, so we any fix in those files now either requires modifying code or merging it first?

Am I misunderstanding this?

Reply:
Trying something like #28782